src/internalUtils/components/ExampleList/ExampleList.tsx

```
@@ -0,0 +1,54 @@
+      if (!maxWidth || maxWidth < newMaxWidth) {
+        maxWidth = newMaxWidth;
+      }
+    });
+
+    setCalculatedWidth(`${maxWidth}px`);
+  }, [cssVariableDetails]);
+
+  return (
+    <ul style={{ listStyle: "none", paddingLeft: "unset" }} ref={exampleListRef}>
+      {cssVariableDetails.map((cssVariableDetail) => (
+        <li id="example-list" style={{ display: "flex", gap: "1rem", alignItems: "center", marginBottom: "0.5rem" }}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)